packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -4,6 +4,7 @@ import {
   BaseControllerV2,
   RestrictedControllerMessenger,
 } from '@metamask/base-controller';
```

Comment:
I don't think changes in this controller are required. Can we just not call `this.currencyRateController.start();` in the metamask-controller if we want this to be disabled, and then control starting and stopping in reponse to preference state changes from within the metamask-controller?

Reply:
`updateExchangeRate` is called internally in the controller from `setNativeCurrency` and `setCurrentCurrency`; I had to add a controlling property to disable and enable the fetch.